### ISSUE TITLE
Please refine documentation of org.openrewrite.java.logging.ParameterizedLogging

### ISSUE THREAD

**Issue by koppor** [openrewrite/rewrite-docs#249]
I try to get https://docs.openrewrite.org/recipes/java/logging/parameterizedlogging running in my project.

I would like to change the parameterization of all log levels... Think, I need to enumarate all of them?

I tried with two, but no change in my code

```yaml
 - org.openrewrite.java.logging.ParameterizedLogging:
     methodPattern: org.slf4j.Logger info(..)
     remoteToString: true
 - org.openrewrite.java.logging.ParameterizedLogging:
     methodPattern: org.slf4j.Logger debug(..)
     remoteToString: true
```

Following statement is still unchanged:

```java
LOGGER.info(String.format("Truncated the too long filename '%s' (%d characters) to '%s'.", fileName, fileName.length(), shortName));
```

---

Maybe, this could also be a wish for a new recipe?

> Replace `String.format` by parameterized logging statements

**Comment by koppor** [comment]
The configuration also does not lead to any change in the code

```java
LOGGER.error("Failed to write XMP data to PDFs:\n" + failedWrittenFiles);
```

**Comment by timtebeek** [comment]
Not quite sure how to best solve this from a documentation perspective. Would it help if we link to places where `...logging.ParameterizedLogging` is used on the documentation page to surface `...logging.slf4j.ParameterizedLogging`?
The former is a building block for the latter; not sure if we recommend use it on it's own outside that SLF4J migration.

/cc @mike-solomon

**Comment by mike-solomon** [comment]
Apologies, but I'm not clear on what the issue is here. The documentation linked to in the original post is auto-generated and not something we'd manually change. 

If some part of it is unclear (such as the description), [feel free to submit a PR which modifies the description of the recipe itself](https://github.com/openrewrite/rewrite-logging-frameworks/blob/main/src/main/java/org/openrewrite/java/logging/ParameterizedLogging.java#L58-L61).

If you have a specific question on how to use this recipe that wasn't answered by @timtebeek above, please feel free to join our [community Slack channel](https://join.slack.com/t/rewriteoss/shared_invite/zt-288thgkl1-S6vny3CqE43c6LJFZyflsQ) and we'd be happy to help there.

**Comment by timtebeek** [comment]
So where things were slightly unclear is that we have two similarly named recipes:
- org.openrewrite.java.logging.slf4j.ParameterizedLogging
- org.openrewrite.java.logging.ParameterizedLogging

The slf4j variant uses the configurable logging variant; but it was not immediately clear to folks reading the logging variant that they might be better of using the slf4j variant.

If you both feel that's sufficiently clear going forward, then indeed no changes necessary.

**Comment by mike-solomon** [comment]
Ahh thanks for the clarification @timtebeek. That's tricky. Maybe we could update the description of those recipes to mention that? I'm not sure it makes sense to change the markdown generator to have specific notes to be added for specific recipes. Or maybe you had another thought about where we'd update the links?

**Comment by koppor** [comment]
Thank you for the feedback. I think, with more time spending to understand the recipes (and possibly ignoring the ordering of the links in the "main" indexes), one should get along.

@mike-solomon Thank you for offering your help above. I tried to focus on getting the SLF4J recipes running again. I posted my issue at https://rewriteoss.slack.com/archives/C01A843MWG5/p1704285725945399 to reduce discussions here. -- I post the link there for interested googlers possibly having the same issue.
